lib/dialects/postgres/connection-manager.js

```diff
@@ -109,18 +110,21 @@ class ConnectionManager extends AbstractConnectionManager {
           return;
         }
         responded = true;
+        debug(`connection acquire`);
```

Comment:
Maybe we should have a debug statement for the connected being setup aswell, so you can compare timestamps.
Then statements for either disconnection, error or successfully connected.

Perhaps include a unique id so it's comparable.
Not that you necessarily need to do this, it can be added later.